xla/service/gpu/gpu_windowed_einsum_handler.cc

```diff
@@ -329,28 +326,27 @@ absl::StatusOr<bool> HandleRsWindowedEinsumLoop(HloComputation* comp,
       changed = true;
     }
   }
+  // If present, move the dequantization of FP8 operands of the dot into the
+  // while loop to allow gemm_rewriter.cc to rewrite into an FP8 Custom
```

Comment:
I should have commented before, but the triton pass, not just gemm_rewriter.cc, can rewrite into an FP8 matmul. Probably should be phrased as 

...to allow the dequantization and dot to be fused into an FP8 matmul, e.g. by gemm_rewriter.cc.